src/effects/presets/effectchainpreset.h

```diff
@@ -40,6 +40,12 @@ class EffectChainPreset {
     double superKnob() const {
         return m_dSuper;
     }
+    void setReadOnly(bool readOnly) {
```

Comment:
Is this ever used for the false case? If not we may consider to remove the parameter.

Reply:
good point, will remove it.
If someone intends to toggle this state in a later implementation the argument may be added again.